pkg/analyse/dashboard.go

```diff
@@ -0,0 +1,36 @@
+package analyse
+
+type (
+	Board struct {
+		ID         uint       `json:"id,omitempty"`
+		UID        string     `json:"uid,omitempty"`
+		Slug       string     `json:"slug"`
+		Title      string     `json:"title"`
+		Panels     []*Panel   `json:"panels"`
+		Rows       []*Row     `json:"rows"`
+		Templating Templating `json:"templating"`
+	}
+	Templating struct {
+		List []TemplateVar `json:"list"`
+	}
+	TemplateVar struct {
+		Name  string      `json:"name"`
+		Query interface{} `json:"query"`
+		Type  string      `json:"type"`
+	}
+	Panel struct {
+		Targets []Target `json:"targets,omitempty"`
+		Title   string   `json:"title"`
+		Panels  []Panel  `json:"panels"` // row panel type
+		Type    string   `json:"type"`
+	}
+	Target struct {
+		RefID      string `json:"refId"`
+		Datasource string `json:"datasource,omitempty"`
```

Comment:
This will not parse modern dashboards, where this is an object. So we still need either https://github.com/grafana/cortex-tools/pull/239 or my change from https://github.com/grafana-tools/sdk/pull/201

Reply:
Ah I don't think this field actually gets used anywhere at the moment, so i could remove it. 

However if we fork, I agree that it makes more sense to just use the fork instead of the above approach.

Reply:
I have a fork at https://github.com/colega/grafana-tools-sdk which you can use here as go mod replacement if you need, I've been merging all the fixes into master there.